### PR TITLE
 gep-2162 - small fixes + bump to experimental

### PR DESCRIPTION
**Pull request by LiorLieberman**
**What type of PR is this?**
/kind cleanup

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by youngnick** [comment]
LGTM, thanks @LiorLieberman.

I'll just approve now though, and wait for further review.

/approve

File: geps/gep-2162.md
Comment (robscott) [inline geps/gep-2162.md:188]: Assuming you meant to add some related content below this heading?

**Review by robscott — COMMENTED** [review]
Thanks @LiorLieberman! I think this may be missing a bit of content but otherwise I think it's ready for experimental.

File: geps/gep-2162.md
Comment (robscott) [reply geps/gep-2162.md:188]: Honestly I think it's also fine to just leave it in future work. But to complete this I think we'd want to document how this feature works + what each entry in the list means. So maybe that would live somewhere like https://gateway-api.sigs.k8s.io/api-types/gatewayclass/. With that said, I think a follow up is fine for that. It just feels awfully funny to have a heading with no content below it as the case is now.

Comment (LiorLieberman) [reply geps/gep-2162.md:188]: added a line to describe the work. 
Hopefully this addresses the concern

**Review by shaneutt — APPROVED** [review]
I think it's time to get this moving into experimental and have implementations try it out :+1:

**Comment by shaneutt** [comment]
Uhhh... I think we should have left the unhold for Rob who still had active comments.

**Comment by robscott** [comment]
> Uhhh... I think we should have left the unhold for Rob who still had active comments.

Reviewed this one retroactively and the updates LGTM after all, but yeah, would be good to leave the hold in place in the future.

**Comment by LiorLieberman** [comment]
> Uhhh... I think we should have left the unhold for Rob who still had active comments.

Uhhh... my bad sorry, i was from the Github App and couldn't figure out where the hold came from.

**Pull request merged**
